test/arrays/ferguson/semantic-examples/0-init-alias.chpl

```diff
@@ -0,0 +1,10 @@
+use samples;
+// var one = new R(1);
```

Comment:
I'm assuming this test only works because samples defines a "one".  If so, should this line be removed?